torch/nn/_functions/thnn/auto_symbolic.py

```diff
@@ -12,7 +12,17 @@ def leakyrelu_symbolic(g, input, negative_slope, inplace=False):
     return g.op("LeakyRelu", input, alpha_f=negative_slope)
 
 
+def glu_symbolic(g, input, dim):
+    assert input.type().sizes()[dim] % 2 == 0
+
+    split_op = g.op('Split', input, axis_i=dim)
+    first = g.createSelect(split_op, 0)
+    second = g.createSelect(split_op, 1)
```

Comment:
I think this could have been just `first, second = g.op('Split', input, axis_i=dim, outputs=2)`

Reply:
Aha definitely better. I was looking for a way to specify the number of outputs.